### PR TITLE
Textarea focus

### PR DESCRIPTION
**Pull request by michaeldebetaz**
The textarea element doesn't gain focus when receiving an error.

My small changes (thanks to the author for his help in making this PR):

1. Fixed the validation process so it includes the textarea element
2. Added testing to check if a textarea component receive focus when the form is invalid. Also included a textarea in the test that checks displayed errors.

I'm really a noob in PR, so let me know if I can do anything better.

### CODE REVIEW

File: apps/test-app/app/components/Textarea.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  label,
+  value,
+  "data-testid": dataTestId,
+}) => {
+  const { error, getInputProps } = useField(name);
+  return (
+    <>
+      <label>{label}</label>
+      <textarea {...getInputProps()} data-testid={dataTestId}>
+        {value}
```
Comment (airjp73) [inline apps/test-app/app/components/Textarea.tsx:22]: I don't think we need to worry about `value` for this component. We can remove this prop entirely.

**Comment by michaeldebetaz** [comment]
Alright, now I got my yarn config to work :) Here is my fixed PR 👍

**Comment by airjp73** [comment]
There were a few tiny changes I went ahead and made to get the tests fully passing, but otherwise, good work. :)

**Comment by michaeldebetaz** [comment]
Many thanks! Sorry for this mess. The idea was to save your time while you were moving 😅

As I would love to contribute again in the future (I'm working with RVF on a medium project), a CONTRIBUTE.md would be very helpful. Right now I'm not sure about how to run the tests that would inform of such errors.

**Comment by airjp73** [comment]
No worries at all! You definitely did save me time.

**Comment by michaeldebetaz** [comment]
Great :) What's next then ? Should I update my package ? Remove and install it ? Or do I wait for I new release ?
I tried reinstalling it, but with no luck. No rush though, it's just I have no idea how this works ^^

**Pull request merged**